core/global_constants.cpp

```diff
@@ -572,8 +572,7 @@ void register_global_constants() {
 	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_USAGE_GROUP);
 	BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_USAGE_CATEGORY);
 	//deprecated, replaced by ClassDB function to check default value
-	//BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_USAGE_STORE_IF_NONZERO);
-	//BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_USAGE_STORE_IF_NONONE);
+
```

Comment:
Unnecessary line. Above comment should also be removed.
And it needs to be removed from other locations too:
```
$ rg PROPERTY_USAGE_STORE_IF_NONZERO
core/object.h
109:    //PROPERTY_USAGE_STORE_IF_NONZERO = 512, //only store if nonzero

core/global_constants.cpp
575:    //BIND_GLOBAL_ENUM_CONSTANT(PROPERTY_USAGE_STORE_IF_NONZERO);

modules/gdnative/include/nativescript/godot_nativescript.h
101:    GODOT_PROPERTY_USAGE_STORE_IF_NONZERO = 512, // FIXME: Obsolete: drop whenever we can break compat
```